addons/host-upgrades/addon.rb

```diff
@@ -1,20 +1,33 @@
 # frozen_string_literal: true
 
 Pharos.addon 'host-upgrades' do
-  version '0.1.0'
+  version '0.0.0'
   license 'Apache License 2.0'
 
   config {
-    attribute :interval, Pharos::Types::String
+    attribute :image_tag, Pharos::Types::String.default("edge")
+    attribute :schedule, Pharos::Types::String
+    attribute :schedule_window, Pharos::Types::String
```

Comment:
Note: The `--schedule-window` defaults to `1h` within `pharos-host-upgrades` itself, which might be a mistake... it should still be possible to disable the `--schedule-window` by using `schedule_window: 0` - or something that makes sense given the `schedule`, like `schedule_window: 24h`.

Reply:
This will now get overridden with `--schedule-window=0` by default, which is IMO good behavior.